src/api/public/apidocs-new/OBS-v2.10.50.yaml

```
@@ -431,6 +432,16 @@ paths:
   /staging/{project_name}/workflow:
     $ref: 'paths/staging_project_name_workflow.yaml'
 
+  # Statistics
+  /statistics/most_active_projects:
+    $ref: 'paths/statistics_most_active_projects.yaml'
+  /statistics/most_active_packages:
+    $ref: 'paths/statistics_most_active_packages.yaml'
+  /statistics/latest_added:
+    $ref: 'paths/statistics_latest_added.yaml'
+  /statistics/latest_updated:
+    $ref: 'paths/statistics_latest_updated.yaml'
```

```suggestion
  # Statistics
  /statistics/latest_added:
    $ref: 'paths/statistics_latest_added.yaml'
  /statistics/latest_updated:
    $ref: 'paths/statistics_latest_updated.yaml'
  /statistics/most_active_packages:
    $ref: 'paths/statistics_most_active_packages.yaml'
  /statistics/most_active_projects:
    $ref: 'paths/statistics_most_active_projects.yaml'
```
We should show the endpoints alphabetically sorted. They will be easier to find.